flybywiresim/aircraft
# feat: add large-files submodule

## Summary of Changes

- adds https://github.com/flybywiresim/aircraft-large-files as submodule
- adjusts build scripts
- adjusts GitHub workflows

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):
foxtrotsierra

## Testing instructions

- download all PR artifacts and check individually if they load in the simulator (different folder name for 8K and 4K version is known and cannot be corrected other than with double-zipping. Change it manually after unzipping, if you care.)
- download all assets (except source code) from https://github.com/flybywiresim/aircraft/releases/tag/assets/master-test and check individually if they load in the simulator
- download all assets (except source code) from https://github.com/flybywiresim/aircraft/releases/tag/assets/stable-test and check individually if they load in the simulator
- download all assets (except source code) from https://github.com/flybywiresim/aircraft/releases/tag/vTest and check individually if they load in the simulator
- download all assets (except source code) from https://github.com/flybywiresim/aircraft/releases/tag/vTest-rc and check individually if they load in the simulator

Be aware you cannot load the A380X 4K and A380X 8K versions simultaneously!


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
